### ISSUE TITLE
Export of members not working

### ISSUE THREAD

**Issue by Scott07uk** [castlamp/zenbership#104]
Get any list of members (ie Members > By Status > Active), click "Export", change nothing (or anything, does not matter), click "Export", nothing happens.

Occurs in Firefox, Internet Explorer and Chrome.  It however does work in Microsoft edge.

**Comment by stevemacguy** [comment]
I have the same problem on a Mac with Chrome.

**Comment by tangdavison** [comment]
I had the same problem. The issue is the popup window closes before the values are pulled from the form, so the php never gets the values. A quick fix is to comment out lines 957 and 958 in admin/js/admin.js. That prevents the pop from closing when the form is submitted. The user can still close the popup in the usual way by clicking the "X" in the top right.

**Comment by jamminjames** [comment]
@tangdavison, your solution is not working for me. I commented out those lines, and nothing has changed, no popup happens, nothing gets exported. Am I missing something? Thanks.

**Comment by steviobee** [comment]
Try another browser? Chrome did not work for me but Edge did on a PC.

**Comment by jamminjames** [comment]
Unfortunately, Edge only works in Windows 10, and for various reasons, I'm not going to go that route for an OS. Doesn't work in Firefox either. Wasn't the idea of tang's fix that it would work on other browsers besides Edge?

**Comment by tangdavison** [comment]
@steviobee, The code works in Edge; we're trying to get it to work in other browsers. Thus, I think this issue should still be open.
@jamminjames, the default functionality was supposed to be, I think, that the popup disappears when the user clicks the "Export" button. What do you mean by "no popup happens." Just so I know we're looking at the same lines of code, here is what I'm commenting out:
`    else if (action == 'close_popup') {
        // Chrome and FF close the popup too fast. The form information
        //  does not get submitted before the popup closes.
        //close_popup(); <--- commented this out.
        //close_large_popup(); <--- commented this out, too.
    }`

**Comment by jamminjames** [comment]
@tangdavison, Yes, I believe I have those same lines commented out. It's at line 957 & 958, as the post said. It's in the function process_success_action(action, additional_data), correct? 
Okay, so I misunderstood, I thought another popup was supposed to happen after hitting the "Export" button.  It's just that it closes too soon, before the export happens? Anyway, even with the lines commented out, I'm still not getting an export of data. How does that export happen, when it happens correctly? Is there anything that comes up, saying save to a folder or something? Also, which data does it export, just what's showing before you hit the Export link? Because that's not much. I'm hoping it will export all the fields for each member requested.

**Comment by jamminjames** [comment]
Also, at line 1350, there's a function:
```
function close_popup() {
    close_large_popup();
   return false;
}
```
... with nothing but some commented out code. It seems like something should be in there, could this be the problem?
